### PR TITLE
BC 75: Last current pagination 

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@projectmushroom @saeedmb8 
Please review 

Before  
screenshot2

After 
screenshot1

File: babycenter/templates/core/section_page.html [hunk 1]
```diff
@@ -18,7 +18,7 @@ <h1 class="heading heading{{self.get_effective_extra_style_hints}} {% if section
 <!-- Not getting translated -->
 
 
-  {% load_child_articles_for_section self count=13 as articles %}
+  {% load_child_articles_for_section self count=3 as articles %}
```
Comment (smarzban) [inline babycenter/templates/core/section_page.html]: @Mitso should the count be 3 here?

Comment (Mitso) [reply babycenter/templates/core/section_page.html]: No, thanks for pointing that out Saeed :)

**Review by projectmushroom — APPROVED** [review]

**Comment by smarzban** [comment]
🍰

**Pull request merged**